app/views/dashboard/_subscription_widget.html.erb

```diff
@@ -1,3 +1,5 @@
+<% require 'erb' %>
```

Comment:
@jlsherrill, yes this work without it, just wanted to be sure that any other user should understand that we are are using ERB at first look, do you want me to remove it?